components/rating/package.json

```
@@ -19,12 +19,12 @@
   },
   "peerDependencies": {
     "@spectrum-css/icon": "^3.0.0",
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^7.1.0"
```

Please update this to point to the latest release.

```suggestion
    "@spectrum-css/tokens": "^7.5.0"
```